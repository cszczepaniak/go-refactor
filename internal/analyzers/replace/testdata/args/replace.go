package basic

import "fmt"

func ReplaceMe(a string, b bool) bool {
	return false
}

func foobar() {
	r := ReplaceMe(fmt.Sprint("abc"), true != false) // want "replace the function call"
	_ = r
}
