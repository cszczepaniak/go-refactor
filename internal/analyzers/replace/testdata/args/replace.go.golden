package basic

import "fmt"

func ReplaceMe(a string, b bool) bool {
	return false
}

func foobar() {
	r := Replaced(true != false, fmt.Sprint("abc")) // want "replace the function call"
	_ = r
}
